### ISSUE TITLE
Add comment [ms3 mass analyzer] and [ms3 dissociation method] for MS3 level?

### ISSUE THREAD

**Issue by Roman-Si** [bigbio/proteomics-sample-metadata#679]
Hi,
I am preparing the sdrf for an LC-MS3 analysis of TMT data. The analysis was done with an Orbitrap Fusion Lumos Tribrid mass spectrometer with the following differences at the MS2 and MS3 level
- MS2: CID 30% and IonTrap
- MS3: HCD 50% and Orbitrap

Following the sdrf documentation and annotated projects I have created the current_sdrf.csv with
- comment[ms2 mass analyzer] to IonTrap
- both CID and HCD as comment[dissociation method and filled comment[CID collision energy] and comment[HCD collision energy] respectively

Would it be helpful to specify the mass analyzer and the dissociation method used for the MS3 as in the proposed_sdrf.csv?
[proposed_sdrf.csv](https://github.com/bigbio/proteomics-sample-metadata/files/12847283/proposed_sdrf.csv)
[current_sdrf.csv](https://github.com/bigbio/proteomics-sample-metadata/files/12847284/current_sdrf.csv)

**Comment by ypriverol** [comment]
@Roman-Si I do like the idea. However, before accepting your proposal we may need to work with the PSI MS team to add the following terms: 

- ms3 mass analyzer
- ms2 mass analyzer
- ms3 dissociation method
- ms2 dissociation method
 
This may take sometime (1 week at least) because I need to do a proposal for the addition of the terms and then, they need to be approved by the PSI-MS team. I will start working on this proposal now.

**Comment by Roman-Si** [comment]
These are great news, thank you!

**Comment by chambm** [comment]
What would you do for a file that alternates HCD and CID for MS2? That's a common case. You can even add ETD in there and have 3 dissociation methods for MS2 spectra.

**Comment by edeutsch** [comment]
I am thinking that the analyzer and dissociation methods are already encoded in the raw or mzML files and the patterns can be simple or very complex within one MS run. Each spectrum is annotated with this information already. As such, I don't think this information should be encoded in the SDRF file. I think we should reserve the SDRF file to encode information that cannot be encoded in the RAW/mzML files, like sample information.

**Comment by ypriverol** [comment]
I think is relevant to save this information in the SDRF. While other formats like mzML, mzQC or others may store this information; we can keep it in the SDRF in the same way we captured the instrument and other parameters. This can serve the following use cases: 

1- Quick searches in repositories, we recently started to index the SDRFs in PRIDE, and we already have multiple users asking about filtering datasets using HCD or CID etc. Then, we are interested in capturing this information. 

2- For automatic reanalysis a lot of tools do need these parameters as input which I think can be useful to have in the SDRF enabling future reanalysis of public data. 

I do think and understand that this may be not a priority use case for PSI-MS, and also it can complicate the Ontology. For that reason, I will move some of this strange/limit use cases to PRIDE ontology as I have been doing before.
